examples/publish-wav/package.json

```diff
@@ -12,11 +12,12 @@
   "keywords": [],
   "license": "Apache-2.0",
   "dependencies": {
-    "@livekit/rtc-node": "workspace:*",
+    "@livekit/rtc-node": "^0.7.0",
```

Comment:
I tried this and tsx couldn't find the library references. do you know what it'll take to make it look up local packages?

the other reason I think it could be useful to link to a released version.. is that our examples will always match published APIs. though if we did that we'd have to automatically bump the version during releases.. probably not ideal

Reply:
did you previously build the package locally at any point? 
invoking tsx with `workspace:*` works for me with no complaints

Reply:
mind editing the readme with the steps that worked for you? i'll verify and update after.

Reply:
Thanks Lukas!

Reply:
trying the build step on my machine, it's taking over 5 mins to compile everything, and running the example failed with:

```
Error: Cannot find module '/Users/davidzhao/dev/livekit/node-sdks/packages/livekit-rtc/node_modules/@livekit/rtc-node-darwin-arm64/rtc-node.darwin-arm64.node'. Please verify that the package.json has a valid "main" entry
    at tryPackage (node:internal/modules/cjs/loader:493:19)
    at Module._findPath (node:internal/modules/cjs/loader:794:18)
    at Module._resolveFilename (node:internal/modules/cjs/loader:1234:27)
```

I'm going back to pointing to latest releases as that's easier for end-users